{{ cookiecutter.directory_name }}/.gitignore

```
@@ -5,3 +5,4 @@
 /.tox
 _trial_temp
 __pycache__
+dist
```

This one feels a bit risky. Is there a reason why this is not good enough?:
```suggestion
/dist
```